include/boost/url/impl/authority_view.ipp

```diff
@@ -114,15 +105,6 @@ encoded_userinfo() const noexcept
     return detail::make_pct_string_view(s);
 }
 
-std::string
```

Comment:
is there any way for us to know the decoded size of the userinfo?

Reply:
Updated.